docs/section6/Rule6-10.md

```diff
@@ -0,0 +1,50 @@
+
+# Lighting - Rule 6-10  
+
+**Rule ID:** 6-10   
+**Rule Description:** Where retail display lighting is included in the proposed building design in accordance with Section 9.5.2.2(b), the baseline building design retail display lighting additional power shall be equal to the limits established by Section 9.5.2.2(b) or same as proposed, whichever is less.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.1 6 Baseline 
+
+**Mandatory Rule:** True  
+**Evaluation Context:** Each Space  
+**Function Call:**  
+
+- get_component_by_id()
+
+## Applicability Check:  
+- look at each building segment: `for building_segment in P_RMD.building_segments:`
+    - get the building segment lighting type: `building_segment_lighting_type = building_segment.lighting_building_area_type`
+    - look at each space: `for space in building_segment...spaces:`
+        - set a boolean called applicable to false: `applicable = false`
+        - set lighting_space_type equal to the space.lighting_space_type: `lighting_space_type = space.lighting_space_type`
+        - if the lighting_space_type is NULL, set the lighting_space_type equal to the building_segment_lighting_type: `if lighting_space_type == NULL: lighting_space_type = building_segment_lighting_type`
+        - if the lighting space type is one of the retail space types, look at each InteriorLighting object in the model: `if lighting_space_type == "SALES AREA":`
+            - look at each interior lighting: `for interior_lighting in space.interior_lighting:`
+                - if the interior lighting purpose_type is RETAIL_DISPLAY, set applicable to true - we don't go to the rule logic for each interior lighting because the evaluation context is at the space level: `if interior_lighting.purpose_type == "RETAIL_DISPLAY": applicable = true`
+        - if the boolean applicable is true, continue to rule logic: `if applicable: CONTINUE TO RULE LOGIC`
+        - otherwise, rule is not applicable: `else: RULE NOT APPLICABLE`
```

Comment:
Maybe I have not got there yet, but I think this rule should also make sure that inapplicable space types are not modeling an extra allowance for retail_display lighting, or do you think this would be picked up as an issue in another rule?